<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWithStudents</class>
 <widget class="QDialog" name="ListWithStudents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainResources.qrc">
    <normaloff>:/Icons/MainResources/PGEE_NEW_LOGO.ico</normaloff>:/Icons/MainResources/PGEE_NEW_LOGO.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="FiltrationLine"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="FiltrationToggle">
       <property name="text">
        <string>Без филтрация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="NamesBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="RefreshAndSaveButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainResources.qrc">
           <normaloff>:/Icons/MainResources/refresh.ico</normaloff>:/Icons/MainResources/refresh.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PrintButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainResources.qrc">
           <normaloff>:/Icons/MainResources/_active_ Print.ico</normaloff>:/Icons/MainResources/_active_ Print.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainResources.qrc">
           <normaloff>:/Icons/MainResources/_active_ No.ico</normaloff>:/Icons/MainResources/_active_ No.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MainResources.qrc"/>
 </resources>
 <connections/>
</ui>
